### PR TITLE
Dcj/feature/light theme

### PR DESCRIPTION
**Pull request by sene-plenty**

### CODE REVIEW

**Review by dpanta-plenty — APPROVED** [review]

**Review by pfrincu-plenty — APPROVED** [review]

File: src/partials/body-scripts.hbs [hunk 1]
```diff
@@ -26,23 +26,31 @@
     }
 
     docsearch({
-      apiKey: '45aeef71ca14d766195975ac179f14d0',
-      indexName: 'plentymarkets',
+      appId: '6PTARL4KJP',
```
Comment (pfrincu-plenty) [inline src/partials/body-scripts.hbs:29]: why did you change this ?

File: src/partials/body-scripts.hbs [hunk 2]
```diff
@@ -26,23 +26,31 @@
...
-        'facetFilters': [
-          ['tags:developers', 'version:' + interface_version, 'version:' + plugin_version]
-        ]
       },
     })
   })()
 </script>
-<script>
+
+{{#if (or (eq site.url 'https://knowledge.plentymarkets.com/de-de') (eq site.url 'https://knowledge.plentymarkets.com/en-gb')) }}
```
Comment (pfrincu-plenty) [inline src/partials/body-scripts.hbs:41]: What happened here?

**Review by pfrincu-plenty — APPROVED** [review]

**Review by ksted — APPROVED** [review]
